### ISSUE TITLE
Read date time cell from Google table

### ISSUE THREAD

**Issue by skynin** [PHPOffice/PhpSpreadsheet#1852]
This is:

```
- [*] a bug report
```

### What is the expected behavior?
I have a google table with column:

2021-02-15 20:30:00  
2021-02-14 23:00:00  
2021-02-15 03:45:00  

$eachCell->getStyle()->getNumberFormat()->getFormatCode() returns:
'yyyy"-"mm"-"dd" "hh":"mm":"ss',

and $eachCell->getFormattedValue() returns:
2021-02-15 20:02:00  
2021-02-14 23:02:00  
2021-02-15 03:02:00  

### My the dirty fix

```php
namespace PhpOffice\PhpSpreadsheet\Style;

class NumberFormat extends Supervisor

    public function getFormatCode() {
        return str_replace(':"mm"', ':"i"', $this->getOriginalFormatCode());
    }
    public function getOriginalFormatCode() // rename original method
    {
        if ($this->isSupervisor) {
            return $this->getSharedComponent()->getFormatCode();
        }
        if ($this->builtInFormatCode !== false) {
            return self::builtInFormatCode($this->builtInFormatCode);
        }

        return $this->formatCode;
    }
```

My another fix didnt help:
```php
class NumberFormat extends Supervisor

private static $dateFormatReplacements = [

        'mmm' => 'M',
        //    mm is minutes if time, but can also be month w/leading zero
        //    so we try to identify times be the inclusion of a : separator in the mask
        //    It isn't perfect, but the best way I know how
        ':mm' => ':i',
        'mm:' => 'i:',
        ':"mm"' => ':"i"', // skynin
        '"mm":' => '"i":', // skynin
```

### Which versions of PhpSpreadsheet and PHP are affected?
"version": "1.16.1"

**Comment by MarkBaker** [comment]
Please try against the latest master branch

**Comment by skynin** [comment]
"version": "1.17.1",

This issue is actual.  
My fix:  
`return str_replace(':"mm"', ':"i"', $this->getOriginalFormatCode());`

$dateFormatReplacements wasn't changed

**Comment by oleibman** [comment]
Fixed by PR #1891
